src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,18 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import core.mate.academy.model.Machine;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Machine> {
```

Comment:
Why do we parameterize MachineProducer by Machine class? (Same in others Produsers)

Reply:
I don't know about I thought when was do it. I need to parameterize a specific machine. BulldozerProducer -> Bulldozer ...